src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const items = [...document.querySelectorAll('[data-salary]')];
+const ul = document.querySelector('ul');
+
+function sortList(list) {
+  return list.sort(
+    (firstItem, secondItem) =>
+      getIterableItems(firstItem) - getIterableItems(secondItem),
+  );
+}
+
+function getIterableItems(item) {
```

Comment:
it works with and return one item/value
```suggestion
function getIterableItem(item) {
```
or better getSortableValue or getSalaryNumber, etc